client/me/help/help-contact/index.jsx

```diff
@@ -344,6 +345,7 @@ module.exports = React.createClass( {
 		const showChatVariation = olark.isUserEligible && olark.isOperatorAvailable;
 		const showKayakoVariation = ! showChatVariation && ( olark.details.isConversing || olark.isUserEligible );
 		const showForumsVariation = ! ( showChatVariation || showKayakoVariation );
+		const showHelpLanguagePrompt = olark.locale !== i18n.getLocaleSlug();
```

Comment:
Some brackets around the conditional would make this easier to understand with all the `=` signs
```js
const showHelpLanguagePrompt = ( olark.locale !== i18n.getLocaleSlug() );
```